pandas/sparse/tests/test_frame.py

```diff
@@ -768,12 +768,20 @@ def _check(frame, orig):
         self._check_all(_check)
 
     def test_count(self):
-        result = self.frame.count()
         dense_result = self.frame.to_dense().count()
+
+        result = self.frame.count()
+        tm.assert_series_equal(result, dense_result)
+
+        result = self.frame.count(axis=None)
+        tm.assert_series_equal(result, dense_result)
+
+        result = self.frame.count(
```

Comment:
you can directly put it here, you don't need to use ``_stat_axis_number`` anywhere (except internallY)